packages/blockchain-wallet-v4-frontend/package.json

```diff
@@ -36,9 +36,10 @@
     "collectCoverageFrom": [
       "src/**/*.js",
       "!src/**/rootSaga.js",
+      "!src/data/components/sagas.js",
```

Comment:
middleware & modules should also be ignored

Reply:
fixed 👍